app/controllers/game.js

```diff
@@ -22,10 +22,14 @@ function validateNameUnique(newName) {
  */
 function getGames(req, res) {
   const fetchBody = {};
+  let knexQuery = Game;
   if (req.query) {
+    // Optional model relations
     if (req.query.withRelated) fetchBody.withRelated = req.query.withRelated;
+    // Filter by storyteller_id
+    if (req.query.storyteller_id) knexQuery = knexQuery.where('storyteller_id', req.query.storyteller_id);
```

Comment:
Append the `where()` clause to the query if storyteller_id query param is present.